core/src/it/scala/org/ensime/intg/DebugTest.scala

```diff
@@ -274,47 +488,50 @@ trait DebugTestUtils {
     expectMsg(TrueResponse)
 
     val vm = VMStarter(config, className)
```

Comment:
Minor point - and it already existed - I'm becoming a big fan of simple case classes for use cases such as this so we have sensible names on _4 (a future), _3, etc.